lightningd/log.c

```diff
@@ -243,6 +246,12 @@ void logv(struct log *log, enum log_level level, const char *fmt, va_list ap)
 	maybe_print(log, l, 0);
 
 	add_entry(log, l);
+
+	if(notifer) {
+		if(level >= LOG_UNUSUAL)
+			notify_warning(log->lr->ld, l);
```

Comment:
Why not put this logic in the caller?   ie. have the log_unusual and log_broken macros set `call_notifier` and the others not?

Reply:
Thank you! Good idea.